src/Stack/Types/Config.hs-boot

```diff
@@ -0,0 +1,5 @@
+module Stack.Types.Config where
```

Comment:
Note: An alternative to using an .hs-boot file is to parameter `NixOpts` over the resolver type. I found the .hs-boot file to be less verbose in the end.